src/rules/no_eval.rs

```diff
@@ -59,23 +63,61 @@ impl<'c> NoEvalVisitor<'c> {
   fn new(context: &'c mut Context) -> Self {
     Self { context }
   }
+
+  fn maybe_add_diagnostic(&mut self, source: &dyn StringRepr, span: Span) {
+    if source.string_repr().unwrap() == "eval" {
+      self.add_diagnostic(span);
+    }
```

Comment:
This `unwrap` seems a kind of dangerous to me as it will likely cause panics in real world. How about writing like this?

```suggestion
    if source.string_repr().as_deref() == Some("eval") {
      self.add_diagnostic(span);
    }
```